src/wavesurfer.ts

```diff
@@ -540,6 +540,11 @@ class WaveSurfer extends Player<WaveSurferEvents> {
     this.options.interact = isInteractive
   }
 
+  /** Update the visual progress */
+  public updateVisualProgress(time = this.getCurrentTime(), isPlaying = this.isPlaying()) {
```

Comment:
I'm reluctant to add a low-level method like this to the public API, especially since it's targeting quite an edge case.

You can easily render wavesurfer w/o playback by providing peaks and duration but no audio. See this example: 
https://wavesurfer.xyz/examples/?0f80252d71c2f4a1ead0a572e1cefad3

It will still respond to `setTime` or `seekTo` but no actual playback will be happening, so you can visually sync it to an external player.